### ISSUE TITLE
Language Server is flagged as malware

### ISSUE THREAD

**Issue by rbhalla** [hashicorp/vscode-terraform#1417]
### Extension

```
2.25.2
```

### VS Code

```
1.66.2
```

### Operating System

```
macOS Ventura 13.3.1 
```

### Terraform Version

```
1.4.6
```

### Steps To Reproduce

Open VS Code, immediately see this.
<img width="453" alt="Screenshot 2023-05-06 at 20 45 50" src="https://user-images.githubusercontent.com/5897045/236643915-9ebb4d85-900f-444b-8d23-e9a4d0155ac2.png">

No features of the extension work.

## Additional context
What I've tried:
- reinstalling terraform
- reinstalling the extension
- restarting

**Comment by dbanck** [comment]
Hi @rbhalla,

I'm sorry that you ran into this problem. As part of the actions to CircleCI's January 3rd, 2023 security alert, HashiCorp revoked the previous signing key for Apple artifacts on April 24th, 2023. [Read more](https://support.hashicorp.com/hc/en-us/articles/13177506317203)

We've released the 2.26.0 extension version that contains an updated binary. But that version requires a minimum VS Code version of 1.67.2 (you're using 1.66.2). So upgrading VS Code should resolve the problem.

If you cannot upgrade, you can download a newer language server version from [releases.hashicorp.com](https://releases.hashicorp.com/terraform-ls/0.31.1/). After downloading the binary, you have to update the extension settings to use it instead of the bundled one:

```
"terraform.languageServer.path": "/Users/dbanck/Downloads/terraform-ls",
```

Let me know if that solves the issue for you.

**Comment by rbhalla** [comment]
Apologies for such a late response. It unfortunately took me a while to upgrade my VS Code, but the problem has gone away. Thank you for your help!
